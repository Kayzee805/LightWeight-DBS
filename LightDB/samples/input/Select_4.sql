SELECT * FROM Courses WHERE Courses.level >= 8 AND Courses.capacity < 100;
