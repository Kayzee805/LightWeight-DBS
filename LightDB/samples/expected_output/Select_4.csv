302,9,203,40
304,11,201,30
305,10,203,35
